timemachine/cpp/src/summed_potential.cu

```diff
@@ -48,9 +57,12 @@ void SummedPotential::execute_device(
             d_du_dx,
             d_du_dp == nullptr ? nullptr : d_du_dp + offset,
             d_u,
-            stream);
+            pot_stream);
 
         offset += params_sizes_[i];
+        if (!serial_) {
```

Comment:
nit: would it be cleaner to call the argument `parallel`? (seems like `serial` mostly appears as `!serial`)

Reply:
Changed to parallel in [78658e3](https://github.com/proteneer/timemachine/pull/1024/commits/78658e3a9a9c8e7d1826020615c9390c7111cfad)